client/flutter/lib/l10n/intl_fr_FR.arb

```diff
@@ -1,68 +1,127 @@
+  "legalLandingPageButtonGetStarted": "Commencer",
+  "legalLandingPageButtonAgree": "En poursuivant, vous acceptez nos",
+  "LegalLandingPageTermsOfServiceLinkText": "Conditions d'utilisation",
+  "legalLandingPageTermsOfServiceLinkUrl": "https://whocoronavirus.org/terms",
+  "legalLandingPageAnd": " et ",
+  "legalLandingPagePrivacyPolicyLinkText": "Politique de confidentialité",
+  "legalLandingPagePrivacyPolicyLinkUrl": "https://whocoronavirus.org/privacy",
+  "@section": "Location Sharing Permission",
+  "locationSharingPageTitle": "Retrouvez toute l'actualité de votre communauté",
+  "locationSharingPageDescription": "Pour accéder aux actualités et informations locales, activez le partage de localisation.",
+  "locationSharingPageButton": "Activer le partage de localisation",
+  "locationSharingPageBackgroundImage": "assets/onboarding/onboarding_location.png",
+  "@section": "Notification Permission",
+  "notificationsPagePermissionRequestPageTitle": "Rester informé(e) sur le Coronavirus",
+  "notificationsPagePermissionRequestPageDescription": "Pour rester au fait de l'actualité sur le Coronavirus, activez les notifications de l'application développée par l'Organisation mondiale de la Santé.",
+  "notificationsPagePermissionRequestPageButton": "Autoriser les notifications",
+  "notificationsPagePermissionRequestBackgroundImage": "assets/onboarding/onboarding_notifications.png",
   "@section": "Home Page",
-  "homePagePageButtonProtectYourself": "Se protéger",
-  "homePagePageButtonLatestNumbers": "Derniers chiffres",
-  "homePagePageButtonYourQuestionsAnswered": "Réponses à vos Questions",
-  "homePagePageButtonWHOMythBusters": "Qui se trompent",
-  "homePagePageButtonWHOMythBustersDescription":"Apprenez les faits sur le Coronavirus et comment prévenir la propagation",
-  "homePagePageButtonTravelAdvice": "Conseils de voyage",
+  "homePagePageTitle": "COVID-19",
+  "homePagePageSubTitle": "Réponse et outils face au virus",
+  "homePagePageButtonQuestions": "Questions & Réponses",
+  "homePagePageButtonProtectYourself": "Protégez-vous",
+  "homePagePageButtonLatestNumbers": "Les derniers chiffres",
+  "homePagePageButtonYourQuestionsAnswered": "Questions & Réponses",
+  "homePagePageButtonWHOMythBusters": "Quels sont les faits",
+  "homePagePageButtonNewsAndPress": "Actualités et presse",
+  "homePagePageButtonWHOMythBustersDescription": "Tout savoir sur le Coronavirus et les mesures barrières pour empêcher la propagation du virus",
+  "homePagePageButtonTravelAdvice": "Conseils aux voyageurs",
   "homePagePageSliverListShareTheApp": "Partager l'application",
-  "homePagePageSliverListProvideFeedback": "Fournir des commentaires sur l'application",
-  "homePagePageSliverListSettingsHeader1": "Collecte de données",
-  "homePagePageSliverListSettingsHeader2": "Langue",
+  "homePagePageSliverListProvideFeedback": "Envoyer des commentaires à propos de l'application",
+  "homePagePageSliverListSettingsHeader1": "Statistiques",
```

Comment:
Statistiques and Data collections are 2 differents things in France. "Statistiques d'usage" or "Collecte de données d'usage" might be more appropriate for GDPR.